### PR TITLE
adds docs for adding a location

### PR DESCRIPTION
**Pull request by punkle**

### CODE REVIEW

File: content/docs/integrations/github-org/index.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
+---
+title: Configuring the GitHub Organization
```
Comment (dtuite) [inline content/docs/integrations/github-org/index.md:2]: I think "Loading Github Organization Teams", like you have in the sidebar, is a better name. "Configuring the GitHub Organization" doesn't tell me why I should be interested in this tutorial.

File: content/docs/integrations/github-org/index.md [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+lastUpdated: '2021-03-19T12:14:39.0Z'
+description: How to configure backstage to read teams from github
+---
+
+## Introduction
+
+You may want to load you organization team structure from github teams.
```
Comment (dtuite) [inline content/docs/integrations/github-org/index.md:9]:
```suggestion
Roadie can load your organization team structure and employees from GitHub teams.
```

File: content/docs/integrations/github-org/index.md [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+This page describes how to do that.
+
+## Prerequisites
+
+- You will need a git repository for the configuration of Backstage. In the examples
+  below, I am calling this backstage-configuration
```
Comment (dtuite) [inline content/docs/integrations/github-org/index.md:16]:
```suggestion
  below, I am calling this backstage-configuration.
```

BTW, as far as I can see, you don't actually refer to this configuration anywhere in the file?

File: content/docs/integrations/github-org/index.md [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+- The github token configured in the backstage secrets must have the read:user and user:email scopes.
```
Comment (dtuite) [inline content/docs/integrations/github-org/index.md:17]:
```suggestion
- The github token configured in the backstage secrets must have the `read:user` and `user:email` scopes.
```

File: content/docs/integrations/github-org/index.md [hunk 5]
```diff
@@ -0,0 +1,40 @@
...
+
+## Steps
+
+1. If the name of your organization was "acme", you would create a file in the github repository called github-org.yaml with the following contents:
+
+```
```
Comment (dtuite) [inline content/docs/integrations/github-org/index.md:23]: You should specify the syntax highlighting here:

    ```yaml

**Review by dtuite — CHANGES_REQUESTED** [review]

**Comment by dtuite** [comment]
@punkle the build failed for an unrelated, intermittent reason. Retriggering should get it to pass. I have an open bug about this #144

File: content/docs/integrations/github-org/index.md (hunk 3 above)
Comment (punkle) [reply content/docs/integrations/github-org/index.md]: removed the reference

File: src/components/doc/Sidebar.js [hunk 6]
```diff
@@ -89,6 +89,11 @@ const DocSidebar = ({ location }) => {
               text="GitHub via Oauth"
               location={location}
             />
+            <SidebarItem
+              to="/docs/integrations/github-org/"
+              text="Loading Github Organization Teams"
```
Comment (dtuite) [inline src/components/doc/Sidebar.js:94]: This label is quite a bit longer than the other items in the sidebar. Can you shorten it to "GitHub teams" and position it underneath the other GitHub related integrations?

Screenshot 2021-03-19 at 14 56 35

File: content/docs/integrations/github-org/index.md [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+## Introduction
+
+Roadie can load your organization team structure and employees from GitHub teams.
+
+This page describes how to do that.
+
+## Prerequisites
+
+- You will need a git repository for the configuration of Backstage. In the examples
+  below.
```
Comment (dtuite) [inline content/docs/integrations/github-org/index.md:16]: "In the examples below". Is this supposed to be a sentence on it's own?

File: content/docs/integrations/github-org/index.md [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+- The github token configured in the backstage secrets must have the `read:user` and `user:email` scopes.
+
+## Steps
+
+1. If the name of your organization was "acme", you would create a file in the github repository called github-org.yaml with the following contents:
+
+```yaml
```
Comment (dtuite) [inline content/docs/integrations/github-org/index.md:23]: The code block has different left margin than the text around it. Either remove the numbers from the steps around this code block or indent the code block so it is visually associated with the step it relates to.

**Review by dtuite — CHANGES_REQUESTED** [review]

File: src/components/doc/Sidebar.js (hunk 6 above)
Comment (punkle) [reply src/components/doc/Sidebar.js]: done

File: content/docs/integrations/github-org/index.md (hunk 8 above)
Comment (punkle) [reply content/docs/integrations/github-org/index.md]: done

**Review by dtuite — APPROVED** [review]

**Pull request merged**